src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -822,24 +822,24 @@ private static String processPageSourceForGodMode(String content) {
                 .replace(TestProperties.inst().TEST_UNREG_ACCOUNT, "${test.unreg}")
                 .replace(Config.SUPPORT_EMAIL, "${support.email}")
                 // today's date
-                .replace(TimeHelper.formatDate(now).replace("/", "&#x2f;"), "{*}")
-                .replace(TimeHelper.formatDate(now), "{*}")
+                .replace(TimeHelper.formatDate(now).replace("/", "&#x2f;"), "${sanitized.today}")
+                .replace(TimeHelper.formatDate(now), "${today}")
                 // now (used in comments last edited date) e.g. [Thu, 07 May 2015, 07:52:13 UTC]
-                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}:[0-9]{2} UTC", "{*}")
+                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}:[0-9]{2} UTC", "${comment.day.month}")
```

Comment:
I had it this way as there were 2 different comments dates. I think #4032 will address the issue. I will change this one